pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!--Please fill in the **bold** fields, and tick all applicable boxes.-->
+
+Added **your URL here**.
+
+**Explain what your list is about and why it is awesome enough to be included here.**
+
+- [ ] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md).
```

Comment:
And a checkbox for having read https://github.com/sindresorhus/awesome/blob/master/create-list.md too